### DISCUSSION TITLE
Q: What is the difference between the two types of "merge LoRA"?

### DISCUSSION THREAD

**Discussion by TomLucidor** [bmaltais/kohya_ss#1751] [Q&A]
1. What is the difference between "**Merge LoRA**" (which allows for optional checkpoints) and "**Merge LoRA (SVD)**" (which allows for "New Rank" and "new Conv Rank")?
2. can LyCORIS models do the same thing with "merging" into a pack?
3. What are the recommendations for "merge precision" and "save precision" based on the LoRAs to be merged?
4. What is "New Rank" and "new Conv Rank" anyway, and given that the LoRA dimensions are known, what are the recommended settings?

**Comment by evaninf** [comment]
7 months and not a single answer?

**Reply by TomLucidor** [reply]
They probably have similar issues to Supermerger when it comes to explaining their algos, not sure https://github.com/hako-mikan/sd-webui-supermerger
